docs/source/java/install.rst

```diff
@@ -0,0 +1,118 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+Installing Java Modules
+=======================
+
+.. contents::
+
+System Compatibility
+--------------------
+
+Java modules are regularly built and tested on macOS and Linux distributions.
+
+Java Compatibility
+------------------
+
+Java modules are currently compatible with Java 8 / 9 / 10 / 11.
+
+Installing from Maven
+---------------------
+
+By default, Maven will download from the central repository: https://repo.maven.apache.org/maven2/org/apache/arrow/
+
+Configure your pom.xml with java module version needed, for example for: memory-netty, format, and vector
```

Comment:
This is something that would be good to explain (though probably elsewhere in the docs): what is the difference between memory-netty and memory-unsafe

Reply:
Yes, this is part of this ticket https://issues.apache.org/jira/browse/ARROW-15573, working on that